docs/source/models/hrnet.md

```diff
@@ -0,0 +1,124 @@
+
+
+# example
+
+outputs['keypoint_scores'] = np.array([[9.93979812e-01, ..., 9.94700551e-01]])
+```
+
+#### Keypoint connections - "keypoint_conns"
+
+A _N_ by _17_ by _2<sub>1</sub>_ by _2<sub>2</sub>_ NumPy array where _N_ represents the number of detected human figures and 17 represents the number of keypoints. 2<sub>1</sub> represents the two connecting keypoints if both keypoints are detected while 2<sub>2</sub> represents the two spatial coordinates of each corresponding connecting keypoint. The two coordinates correspond to:
+
+- x: x-coordinate
+- y: y-coordinate
+
+The order of the keypoint connections corresponds to the order of "bboxes", "keypoints" and 'keypoint_scores'.
+
+```python
+outputs['keypoint_conns'] = np.array([[[[x,y], [x,y]],
+                                       [[x,y], [x,y]],
+                                        ...,
+                                       [[x,y], [x,y]]]])
+
+
+# example
+
+outputs['keypoint_conns'] = np.array([[[[0.37138409, 0.98567304], [0.55192859, 0.59019476]],
+                                       [[0.4532299, 0.58970387], [0.50471611, 0.63052403]],
+                                       ...,
+                                       [[0.4532299, 0.58970387], [0.50471611, 0.63052403]]]])
+```
+
+## Configurable parameters
+
+The full list of configurable parameters for the HRNet node are listed in [hrnet.yml](https://github.com/aimakerspace/PeekingDuck/blob/dev/peekingduck/configs/model/hrnet.yml). To change the default parameters, follow the instructions to configure node behaviour in the [main readme](https://github.com/aimakerspace/PeekingDuck). Below is an example of how the HRNet node is configured to increase the score threshold.
+
+```yaml
+# Example: Configure HRNet node to increase the score threshold
+nodes:
+  - input.live
+  - model.yolo
+  - model.hrnet:
+      - score_threshold: 0.4
+  - draw.poses
+  - output.screen
+```
+
+The table shown below is a list of commonly adjusted parameters for the HRNet node.
+
+| Parameter       | Description                                                                                                                                                                     | Variables                                                                  | Default                         |
+| --------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------- | ------------------------------- |
+| resolution      | The size of the image is resized and padded to before feeding into the model. A larger size will yield higher accuracy while decreasing the speed of inference. When specifying | { height: _int_, width: _int_ }<br> <br>where int must be divisible by 32. | { height: _192_, width: _256_ } |
```

Comment:
Remove the "when specifying" if not needed

Reply:
My bad